electrum/storage.py

```diff
@@ -100,6 +98,18 @@ def write(self, data: str) -> None:
         self._file_exists = True
         self.logger.info(f"saved {self.path}")
 
+    def append(self, data: str) -> None:
+        """ append data to encrypted file"""
+        if self.is_encrypted():
+            self.decrypted += data
+            self.write(self.decrypted)
+            return
```

Comment:
Indeed, since we currently do not have encrypted partial writes, we can just call `_write` if the file is encrypted. 
Added this to condition in `JsonDB.write`